README.md

```diff
@@ -30,5 +30,9 @@ We welcome open source contributions as well!
 
 ## Releases:
 
- - 0.1.0 initial release
- - 0.2.0 release for Spark 2.0 (work of @felixcheung)
+- 0.1.0 initial release
+- 0.2.0 release for Spark 2.0 (work of @felixcheung)
+- 0.3.0
+  - DataFrame-based connected components implementation
+  - added support for Python 3
```

Comment:
I see there was a PR to address this in #135, but isn't this also dependent on https://github.com/databricks/sbt-spark-package/issues/26?